parsers/base64/locales/en-US/70-base64.json

```diff
@@ -0,0 +1,24 @@
+{
+  "base64": {
+    "base64": "base64",
```

Comment:
`base64.base64` seems not to be used (I think that it is node name).

Reply:
Other nodes have similar key/value.
I think it might be used for some purposes in future (e.g. caption of node in palette).

Reply:
Okay. I learned it!